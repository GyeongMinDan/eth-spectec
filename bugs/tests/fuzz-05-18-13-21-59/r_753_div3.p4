control c(inout bit<8> a) {
         bit<8> b = 3;
         apply {
            a = a / b;
        }
     }
