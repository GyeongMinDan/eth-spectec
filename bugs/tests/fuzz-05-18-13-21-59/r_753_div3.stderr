likely_hits/r_753_div3.p4(1): [--Wwarn=unused] warning: Control c is not used; removing
control c(inout bit<8> a) {
        ^
[--Wwarn=missing] warning: Program does not contain a `main' module
